Sources/App/SlackCommand+Options.swift

```diff
@@ -0,0 +1,18 @@
+import Foundation
+import Vapor
+import Stevenson
+
+extension SlackCommand {
+    enum Option {
+        static let branch = "branch"
+        static let repo = "repo"
+    }
+
+    static func branch(fromOptions options: [String]) -> String? {
```

Comment:
Not a hard requirement for now so maybe not for this PR, but might be worth generalizing the concept by refactoring this to `option(_ option: Option, from components: [String]) -> String?` (function name to be bikeshed) and change the call site from `branch(fromOptions: components)` to `option(.branch, from: components)`

Reply:
yeah, though ATM we don't need anything but `branch`, options are being passed to the CI as a plain string. I have an experimental branch that integrates [cli-mate](https://github.com/ilyapuchka/cli-mate) but it is very experimental and I doubt we really need it